cookbooks/kinematically_driven_subduction_2d/kinematically_driven_subduction_2d_case2a.prm

```diff
@@ -37,21 +37,19 @@ subsection Material model
   set Model name = multicomponent
 
   subsection Multicomponent
-    # TODO the benchmark uses different reference temperatures for each composition
-    #set Reference temperature         = 1573.15,473.15,473.15,473.15,473.15,1573.15,1573.15,1573.15
     set Reference temperature         = 473.15
-    set Viscosity averaging scheme    = maximum composition
+    set Viscosity averaging scheme    = maximum composition 
     set Viscosities                   = 1.e20, 1.e23, 1.e20, 1.e23, 1.e23, 1.e23, 1.e23, 1.e20
-    set Densities                     = 3200.0,3000.0,3000.0,3250.0,3250.0,3200.0,3200.0,3200.0
-    set Thermal conductivities        = 183.33,2.5,2.5,2.5,2.5,2.5,2.5,2.5
+    set Densities                     = 3200.0,3250.0,3250.0,3250.0,3250.0,3250.0,3250.0,3250.0
+    set Thermal conductivities        = 183.33,2.5,2.5,2.5,2.5,2.5,2.5,2.5 
     set Heat capacities               = 1250.0,750.0,750.0,750.0,750.0,1250.0,1250.0,750.0
     set Thermal expansivities         = 0
   end
 
 end
 
 # The overriding plate (OP) and subducting plate (SP)
```

Comment:
Can these lines for composition be deleted because they are also the included prm file for Case 1?